docs/getting-started/which-permissions-does-codacy-need-from-my-account.md

```diff
@@ -63,15 +63,15 @@ If you log in with GitHub, Codacy requires the following [app permissions](https
       <td>Read & Write</td>
       <td>Codacy sets the status of commits according to the result of code analysis.</td>
     </tr>
-    <tr>
-      <td>Administration</td>
-      <td>Read & Write</td>
-      <td><a href="#why-does-codacy-ask-for-permission-to-create-ssh-keys">Codacy creates an SSH key</a> on the repository to allow cloning and integrating with your repository.</td>
-    </tr>
     <tr>
       <td>Contents</td>
       <td>Read-Only</td>
-      <td>Codacy accesses repository contents to provide faster code coverage analysis and as part of an initiative to use GitHub App tokens instead of SSH keys when cloning repositories for code quality analysis.</td>
+      <td>Codacy retrieves repository contents to get installation access tokens, which are used to integrate with your repositories and clone them for code analysis.</td>
```

Comment:
I believe we could also include `code coverage` in the first line (I'll be honest, I don't fully understand how and why the new coverage pipeline needs this permission, but it does)

I'd also suggest adding an emphasized call to action here for people to approve this permission.
> Codacy is planing on deprecating the need for Repository Administration permissions, but to do so resources from Contents permssion are needed. This Permission has strated being requesteted recently so to future-proof your codacy usage, make sure you that you aprove [approve Codacy GitHub App updated Repository Coontents permission](https://docs.github.com/en/apps/using-github-apps/approving-updated-permissions-for-a-github-app) on your GitHub organization.